app/container.php

```diff
@@ -143,6 +144,9 @@ function (ConsoleCommandEvent $event) use ($userInteraction) {
 $container['command.stop'] = function ($c) {
     return new StopCommand($c['project.manager'], $c['compose.project']);
 };
+$container['command.build'] = function ($c) {
+    return new BuildCommand($c['project.manager.docker_compose'], $c['compose.project']);
```

Comment:
You probably want to either inject `project.manager` or an other service (that can/should be an alias) called `project.build_manager`.